cli/db_backup.go

```diff
@@ -0,0 +1,128 @@
+package cli
+
+import (
+	"bufio"
+	"errors"
+	"math"
+	"os"
+	"path/filepath"
+
+	"github.com/dgraph-io/badger/v3"
+	"github.com/spf13/cobra"
+)
+
+var (
+	dir         string
+	backupFile  string
+	numVersions int
+	restoreFile string
+)
+
+func init() {
```

Comment:
would be better to structure this similar to existing CLIs (e.g. `export_keys_from_mnemonic.go`, `generate_operator_keys.go` etc.):
- use a single file per CLI, could be under a new package `ops` or `db`
- use cobra.Command to read flags into local vars (in the function level)